src/storybook/components/usage-guidelines/usage-guideline.scss

```diff
@@ -4,16 +4,23 @@
 .monday-storybook-usage-guidelines{
   margin-top: $spacing-between-section-items;
   &_guideline {
-    margin: $spacing-medium 0;
+    color: var(--primary-text-color);
+    margin: var(--spacing-medium) var(--spacing-small);
     display: flex;
     align-items: center;
+    font-size: 16px;
```

Comment:
we usually using line-height: 24px in other text components. is the guideline should be different?
also there are mixins for text in the storybook new design (storybook/styles/typography) - but there we using 24 and not 21 line height.